src/kafka/lsst.rs

```diff
@@ -40,7 +40,7 @@ impl AlertConsumer for LsstAlertConsumer {
         if self.simulated {
             "alerts-simulated".to_string()
         } else {
-            "alerts".to_string()
+            "lsst-alerts-v9.0".to_string()
```

Comment:
Maybe worth making configurable?

Reply:
Yeah. My new consumer CLI has a subcommand for each survey (`decam`, `lsst`, `ztf`), and each subcommand has whatever args it needs to get the topic name. For LSST, there's an arg for the schema version. For ZTF and DECam, there's an arg for the date.